tests/test_workers/test_greenwave.py

```diff
@@ -0,0 +1,101 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import json
+from unittest import mock
+
+import pytest
+
+from iib.workers import greenwave
+from iib.exceptions import IIBError
```

Comment:
Optional: It's nice to keep the imports alphabetized.

Reply:
Done :)